docs/userguide.md

```diff
@@ -504,6 +504,8 @@ the `orphaning-deletion` by:
  kubefedctl orphaning-deletion disable <federated type> <name>
  ```
 
+If the flag `--namespace` is additionally not specified, the <federated resource> will 
```

Comment:
Please remove the angle brackets from federated resource as per the example of other instances in this paragraph.

Reply:
Done, I was not sure about `<target resource>`, meantime left it as is.